css/utilities/text-align.css

```diff
@@ -0,0 +1,3 @@
+.text-align-ctr {
```

Comment:
could be just `text-center` I feel.

Reply:
We can do this, I think. Thanks, I will change.